### PR TITLE
Hometask 9

### PR DESCRIPTION
**Pull request by pu6karev**

### CODE REVIEW

File: bank-manager-server/src/main/java/com/hillel/multi/service/AccountService.java [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
+//
+//    public List<AccountDTO> getAllAccounts(){
+//        return  null;
+//    }
+//
+//    public List<AccountDTO> getAccountsByCustomer(Integer customerId){
+//        return null;
+//    }
+//
+//    public AccountDTO getAccountById(Integer accountId){
```
Comment (Vedmider) [inline bank-manager-server/src/main/java/com/hillel/multi/service/AccountService.java]: What is this ? What for ?

File: bank-manager-server/src/main/java/com/hillel/multi/service/CustomerService.java [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
+//    public List<CustomerDTO> getAllCustomers(){
+//        return null;
+//    }
+//
+//    public CustomerDTO getCustomerById(Integer customerId){
+//        return null;
+//    }
+//
+//    public CustomerDTO createCustomer(CustomerDTO customer){
+//        return null;
```
Comment (Vedmider) [inline bank-manager-server/src/main/java/com/hillel/multi/service/CustomerService.java]: Same here

File: pom.xml [hunk 3]
```diff
@@ -109,6 +110,20 @@
                 <plugin>
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>3.11.0</version>
```
Comment (Vedmider) [inline pom.xml]: Please put version to properties

File: bank-manager-server/src/main/java/com/hillel/multi/service/AccountService.java [hunk 4]
```diff
@@ -1,34 +1,34 @@
...
+//
+//import java.util.List;
+//
+//@Service
+//public class AccountService {
+//
+//    public List<AccountDTO> getAllAccounts(){
+//        return  null;
+//    }
+//
```
Comment (Vedmider) [inline bank-manager-server/src/main/java/com/hillel/multi/service/AccountService.java]: Commented classes should not be

Comment (pu6karev) [reply bank-manager-server/src/main/java/com/hillel/multi/service/AccountService.java]: I've deleted these classes in new commit

**Pull request merged**